index.html

```diff
@@ -2,80 +2,1625 @@
+            </div>
+            <div class="card-body p-0 mt-2 mb-4">
+              <h5 class="card-title font-weight-bold">2019 College Football Schedule: Dates, TV Channels, Live Stream,
+                Odds</h5>
+            </div>
+          </div>
+        </article>
+      </div>
+
+      <div class="col-md-4">
+        <div class="sub-section-picks">
+          <ul class="pl-0">
+            <li class="d-flex">
+              <img
+                src="https://d.newsweek.com/en/full/1519856/san-francisco-49ers.jpg?w=135&h=90&f=1e4e394a3aeabf06c6c2f530661e9194"
+                alt="NFL Preseason: Where to Watch 49ers vs. Broncos, TV Channel, Live Stream"
+                class="img-thumbnail border-0 mr-2">
+              <div class="sub-info">
+                <span>NFL Preseason: Where to Watch 49ers vs. Broncos, TV Channel, Live Stream</span>
+              </div>
+            </li>
+            <li class="d-flex">
+              <img
+                src="https://d.newsweek.com/en/full/1520761/marcus-rashford-manchester-united.jpg?w=135&h=90&f=a9f41fd54f8ccc14e96336d330300270"
+                alt="Wolves vs. Manchester United: Where to Watch, Live Stream, Team News"
+                class="img-thumbnail border-0 mr-2">
+              <div class="sub-info">
+                <span>Wolves vs. Manchester United: Where to Watch, Live Stream, Team News</span>
+              </div>
+            </li>
+            <li class="d-flex">
+              <img
+                src="https://d.newsweek.com/en/full/1521085/little-league-world-series.jpg?w=135&h=90&f=895345dae5c4e6cff2867a0b658ecd62"
+                alt="Little League World Series: Updated Bracket, TV Schedule" class="img-thumbnail border-0 mr-2">
+              <div class="sub-info">
+                <span>Little League World Series: Updated Bracket, TV Schedule</span>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+
+    </div>
+
+    </div>
+  </section>
+
+  <!-- End Of Sports  -->
+
+  <!------------Subscribe------------->
+
+  <section class="subscribe">
+    <div id="subnews-block" class="position-relative border-bottom">
+      <div class="row">
+        <div class="col-lg-8">
+          <div class="sb-title text-uppercase">subscribe</div>
+          <div
+            class="subs-block d-flex flex-column align-items-center justify-content-center position-relative flex-md-row">
+            <div class="subs-image">
+              <div class="mag-thumb prev">
+                <img
+                  src="https://d.newsweek.com/en/full/1516486/cover-august-09-2019.jpg?w=170&q=95&f=23cfe68d3e6aaeba28a49957c89b1d3e"
+                  alt="">
+              </div>
+              <div class="mag-thumb current">
+                <img
+                  src="https://d.newsweek.com/en/full/1518227/cover-august-16-23-2019.jpg?w=170&q=95&f=350d013b3af6e3980dbaccd062943428"
```

Comment:
- [ ] This section also looks quite different. 

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88110335-69d56080-cb82-11ea-8945-525e9fa8ac6e.png) | ![image](https://user-images.githubusercontent.com/61479648/88110392-87a2c580-cb82-11ea-8c40-a9a73d6cb439.png) |